src/FedSrv/fsmission.h

```diff
@@ -11,6 +11,10 @@
 #ifndef _FSMISSION_H_
 #define _FSMISSION_H_
 #include "pch.h"
+#include "json.hpp"
+#include "../Lib/cpp-httplib/httplib.h"
```

Comment:
Any reason to use this instead of the existing MaClient (mbedthis HTTP client)?

Reply:
mbedthis doesn't allow changing the encoding header and by default it uses urlencoded while we want json
additionally the source code for it doesn't seem to be available anymore and some of the functions in client.h are not included in the .lib file for it
i actually want to replace mbedthis everywhere else too to not have 2 libraries doing the same thing

Reply:
Got it, thank you! Will you be able to complete the replacement as part of this work?

Reply:
yeah i've created a separate pull request for it which only needs adding debug logging and testing if everything works. i can merge it into this one if that would be better.

Reply:
I've always wondered why Windows [wininet](https://learn.microsoft.com/en-us/windows/win32/wininet/http-sessions)  wasn't used for MaClient ?

Reply:
[this page](https://learn.microsoft.com/en-us/windows/win32/wininet/wininet-vs-winhttp) says that wininet cannot be used inside a service and fedsrv as well as lobby can be set up as a service. other that that i have no idea though, you could ask imago but he might not remember either